spec/datastore_api/requests/documents/delete_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
```

Comment:
Wonder if there are other tests we can apply this to. May make them slightly faster, as long as we are using them in an immutable way to begin with.